specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -6776,6 +6912,251 @@
       },
       "type": "object"
     },
+    "ShayEntityQueryProperties": {
```

Comment:
Shay?

Reply:
Thanks for catching this. While merging with your code I figure out that and I both used the same names for some of our data models, so I temporary change my to Shay. Fixed @ori-licht